src/main/java/uk/gov/ons/census/casesvc/service/UacProcessor.java

```diff
@@ -111,13 +133,19 @@ public void emitUacUpdatedEvent(UacQidLink uacQidLink, Case caze, boolean active
       uac.setRegion(caze.getRegion());
     }
 
-    Payload payload = new Payload();
-    payload.setUac(uac);
+    PayloadDTO payloadDTO = new PayloadDTO();
+    payloadDTO.setUac(uac);
     ResponseManagementEvent responseManagementEvent = new ResponseManagementEvent();
-    responseManagementEvent.setEvent(event);
-    responseManagementEvent.setPayload(payload);
+    responseManagementEvent.setEventDTO(eventDTO);
+    responseManagementEvent.setPayloadDTO(payloadDTO);
 
     rabbitTemplate.convertAndSend(
         outboundExchange, UAC_UPDATE_ROUTING_KEY, responseManagementEvent);
+
+    return payloadDTO;
+  }
+
+  private String convertPayloadDTOToJson(PayloadDTO payloadDTO) throws JsonProcessingException {
+    return objectMapper.writeValueAsString(payloadDTO);
```

Comment:
I recommend that you catch the `JsonProcessingException` and throw a `RuntimeException`. It's incredibly unlikely that we'd be unable to marshall the object to JSON and having a checked exception being thrown out of this method is not helpful or useful.

Reply:
done